### ISSUE TITLE
Setting path for GunicornPrometheusMetrics

### ISSUE THREAD

**Issue by jackdreilly** [rycus86/prometheus_flask_exporter#26]
I am wondering whether there would be any negative side-effects to setting `GunicornPrometheusMetrics().path` to not be `None`.

I prefer to service the metrics inside of the main `Flask` app, rather than its own `http` server.  

Is there any reason that setting `path` was explicitly disabled in the `GunicornPrometheusMetrics` constructor?

**Comment by rycus86** [comment]
Hi,

I think there was one, but can't remember off the top of my head. Something about multiprocessing?
Not really sure at the moment, but I'll try to test it a bit to see if it can be done.

Thanks!

**Comment by rycus86** [comment]
OK, it looks like it's working fine. :man_shrugging: 
I think I may have started with uWSGI where it needs to be separate (I think), then I assumed it needs to be the same for Gunicorn as well.

In any case, I added `GunicornInternalPrometheusMetrics` in version `0.8.1` (to be released in a few minutes), which accepts a `path` and does what you wanted. I added some notes on it in the README, but similar configuration, minus the start HTTP server.

Let me know how it works out! :)

Thanks!
